persistence/sql/persister_identity.go

```diff
@@ -157,6 +157,14 @@ func createRecoveryAddresses(ctx context.Context, tx *pop.Connection, i *identit
 	return nil
 }
 
+func (p *Persister) CountIdentities(ctx context.Context) (int64, error) {
+	count, err := p.c.WithContext(ctx).Count(new(identity.Identity))
```

Comment:
Counts can be very expensive to compute as the table size grows. A caching layer might serve well in the future for this kind of use case but that would require an across the board tuning once that scale is hit.

Reply:
Thank you, that's definitely true. There are ways to improve that though (e.g. with caching). There are some PRs open in gobuffalo to help with this.